corehq/apps/locations/permissions.py

```diff
@@ -95,7 +95,7 @@
     "This project has restricted data access rules. Please contact your "
     "project administrator to be assigned access to data in this project. "
     'More information is available <a href="{link}">here</a>.'
-).format(link="https://wiki.commcarehq.org/display/commcarepublic/Data+Access+Restrictions"))
+).format(link="https://wiki.commcarehq.org/display/commcarepublic/Data+Access+and+User+Editing+Restrictions"))
```

Comment:
already addresed in https://github.com/dimagi/commcare-hq/pull/14528/, though seems to merge cleanly.